src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -39,6 +39,7 @@ public partial class ZipArchiveEntry
         private bool _currentlyOpenForWrite;
         private bool _everOpenedForWrite;
         private Stream _outstandingWriteStream;
+        private uint _externalFileAttr;
```

Comment:
It doesn't really matter. We could use an int internally instead, I just like the uint since you don't have to worry about bit carrying when doing shifts and it's already being using in the ZipCentralDirectoryFileHeader reader. Both could be changed to ints easily though.